frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -92,6 +100,32 @@ pub mod pallet {
 			VaultId = Self::VaultId,
 		>;
 
+		/// Currency is used for the assets managed by the vaults.
+		type Currency: Transfer<Self::AccountId, Balance = Self::Balance, AssetId = Self::AssetId>
+			+ Mutate<Self::AccountId, Balance = Self::Balance, AssetId = Self::AssetId>
+			+ MutateHold<Self::AccountId, Balance = Self::Balance, AssetId = Self::AssetId>;
+	
+		type Pablo: Amm<
+			AssetId = Self::AssetId,
+			Balance = Self::Balance,
+			AccountId = Self::AccountId,
+			PoolId = Self::PoolId,
+		>;
+
+		// Type representing the unique ID of a pool.
```

Comment:
If it is docs, than the line should start with `///` (three times slash), if not, than you can free to resolve the comment.